docs/source/model-registry.rst

```diff
@@ -81,7 +81,7 @@ version to one of the other valid stages.
 API Workflow
 ============
 
-An alternative way to interact with Model Registry is using the `MLflow model flavor <https://www.mlflow.org/docs/latest/python_api/index.html>`_ or `MLflow Client Tracking API <https://www.mlflow.org/docs/latest/python_api/mlflow.tracking.html>`_ interface.
+An alternative way to interact with Model Registry is using the :ref:`MLflow model flavor <python-api>` or :ref:`MLflow Client Tracking API <mlflow.tracking>` interface.
```

Comment:
Thanks for doing these docs cleanups. I verified them locally -- they work great. Awesome cleanup.